src/routing/Router.svelte

```diff
@@ -54,7 +54,7 @@
       if (probeName) {
         store.setField('probeName', probeName);
         if ($probeSet) {
-          let newProbe = $probeSet.find((p) => p.name === probeName);
+          let newProbe = $probeSet.find((p) => p.name.toLowerCase() === probeName.toLowerCase());
```

Comment:
Do we need to compare against `probeName.toLowerCase()`? We don't support non-lowercase probe names in the URL and to avoid canonicality issues I don't think we should.

http://localhost:5000/firefox/probe/gC_mS/explore?activeBuckets=%5B%22no%22%2C%22yes%22%5D&aggregationLevel=build_id&channel=nightly&os=Windows&process=content&proportionMetricType=proportions&timeHorizon=MONTH&visiblePercentiles=%5B95%2C75%2C50%2C25%2C5%5D